Makefile

```diff
@@ -17,7 +17,7 @@ proto: scripts/protocc/protocc.py
 # TODO(nick): We should have a release build that bakes in a particular
 # SYNCLET_IMAGE tag.
 install:
-	./hide_tbd_warning go install ./...
+	./hide_tbd_warning go install -ldflags "-X 'github.com/windmilleng/tilt/internal/cli.version=$$(<dev_version)-dev'" ./...
```

Comment:
again, i think this line misunderstands the problem that this variable is trying to solve. it's for people that don't use our Makefile